October 2023 Meetup Post (#1456)
rust-gamedev/rust-gamedev.github.io: content/blog/rust-gamedev-meetup-31/index.md

+++
title = "Rust Gamedev Meetup 31"
date = 2023-10-14
transparent = true
aliases = ["posts/gamedev-meetup-31"]
+++

<!-- markdownlint-disable single-title heading-increment -->
<!-- markdownlint-disable no-blanks-blockquote no-emphasis-as-header -->
<!-- markdownlint-configure-file {"line-length": {"heading_line_length": 120}} -->

![Rust Gamedev Meetup](gamedev-meetup.png)

The 31th Rust Gamedev Meetup took place on October 14th. You can watch the
recording of the meetup [here on YouTube][meetup-video]. The meetup will take
place on the Rust Gamedev [YouTube][youtube-stream] and [Twitch][twitch-stream].
The meetups take place on the second Saturday of every month via the [Rust
Gamedev Discord server][rust-gamedev-discord] and are also [streamed on
Twitch][rust-gamedev-twitch].

[rust-gamedev-discord]: https://discord.gg/yNtPTb2
[rust-gamedev-twitch]: https://twitch.tv/rustgamedev
[meetup-video]: https://www.youtube.com/watch?v=MadryxOwjb8
[youtube-stream]: https://www.youtube.com/@RustGameDevelopment
[twitch-stream]: https://www.twitch.tv/RustGameDev

## Talks

### [Games, but no video: Embedded trinkets in Rust] | [@Noxim]

![Hardware](hardware.jpg)

> > An overview of making games with constrained hardware

Author: @noxim What would go better as a talk description?

- [Slides]
- [Hardware]
- [Firmware]
- [Rustc]
- [PAC + HAL]
- [Blog post][More details]

[Slides]: https://docs.google.com/presentation/d/1LnxSBdzKOGLGbUAiyaZShYfjJYitCOkM8MzPNLfFEQc/edit?usp=sharing
[Hardware]: https://oshwlab.com/noxim/nxr-w-101-001_copy
[Firmware]: https://owo.codes/noxim/nxr-t202-bitfiddle
[Rustc]: https://github.com/noxime/rust/tree/rv32e-upstream
[PAC + HAL]: https://github.com/ch32-rs
[More details]: https://noxim.xyz/blog/rust-ch32v003/
[Games, but no video: Embedded trinkets in Rust]: https://www.youtube.com/live/MadryxOwjb8?si=erY49m_5_HDFkrpf&t=694
[@Noxim]: https://github.com/Noxime

### [Tunnet] | [@puzzled_squid]

![tunnet title](tunnet.jpg)

> > A short game where you wake up as a robot technician in an underground complex

Author: @puzzled-squid What would go better as a talk description?

Reviewer: I would phrase it this way: "A short game where you design, build, and optimize a computer network in an undergound complex"

> > A short game where you wake up as a robot technician in an underground complex

Author: ```suggestion
> A short game where you design, build, and optimize a computer network in an underground complex
```

> > A short game where you design, build, and optimize a computer network in an underground complex

Author: ```suggestion
> A short game where you design, build, and optimize
> a computer network in an underground complex
```

- [itch.io page]
- [YouTube trailer]
- [Website]

![tunnet title](tunnet-network.jpg)

[itch.io page]: https://puzzled-squid.itch.io/tunnet
[YouTube trailer]: https://www.youtube.com/watch?v=6gxytilYoOM
[Website]: https://puzzledsquid.xyz/
[Tunnet]: https://www.youtube.com/live/MadryxOwjb8?si=vZrhqUszuv9qvG9t&t=2878

> [@puzzled_squid]: https://github.com/puzzled-squid

Author: ```suggestion
[@puzzled_squid]: https://github.com/puzzled-squid

```
